Core/PaymentMethodHelper.php

```diff
@@ -203,4 +378,53 @@ public static function prepareCreditorName()
 
         return $sCreditorName ? $sCreditorName : $oShop->oxshops__oxcompany;
     }
+
+    /**
+     * Checks the user data if mandatory fields are set correctly for guaranteed invoice and saves them if needed
+     *
+     * @since 1.2.0
+     */
+    public static function checkPayStepUserInput()
```

Comment:
I think we have something like that for eps and giropay. Is there somehting in the controller for this!?

Reply:
not yet, but I guess we have to refactor it later

Reply:
moved to payment method